### ISSUE TITLE
Can SALT be used as SCCM replacement

### ISSUE THREAD

**Issue by AmbicaY** [saltstack/salt-winrepo-ng#1315]
Hi

I am trying to evaluate if we can use SALT as a replacement for SCCM. Has anyone tried or compared these 2? Any pointer/inputs will be very helpful.

**Comment by discentem** [comment]
You probably want to join the Saltstack community Slack and discuss this in the #windows windows channel

**Comment by prog-dale** [comment]
Yes I have and I am in the process of replacing all SCCM installations with salt starting with windows patching.

**Comment by discentem** [comment]
I'd definitely be interesting in seeing how you manage windows patching in salt. Haven't tackled that yet (we are currently patching with other tools)

**Comment by DaveQB** [comment]
How did this go @prog-dale ?
